integration_tests/run_pyspark_from_build.sh

```diff
@@ -258,19 +258,25 @@ else
 
     # If you want to change the amount of GPU memory allocated you have to change it here
     # and where TEST_PARALLEL is calculated
-    export PYSP_TEST_spark_rapids_memory_gpu_allocSize='1536m'
+    export PYSP_TEST_spark_rapids_memory_gpu_allocSize=${PYSP_TEST_spark_rapids_memory_gpu_allocSize:-'1536m'}
```

Comment:
Why do we need this change?  The point of this was to set an amount that was always consistent no matter what GPU we run on.  

As a separate note the calculations for TEST_PARALLEL were not changed to take this into account.

https://github.com/NVIDIA/spark-rapids/blob/814732c13c3621698d9b29deff7c70b7cd263b62/integration_tests/run_pyspark_from_build.sh#L124

If you really want this to be configurable then we need to either error out if TEST_PARALLEL is not set and gpu_allocSize is, or we need to make sure that the TEST_PARALLEL calculation take the allocSize into account properly.

I personally see no reason to make this configurable.

Reply:
I did not try to make the magic formula work for my system. As is, my system freezes when I configure a fully distributed setup as in the description: 2 pytest apps with 2 executors each. On my notebook GPU I typically have 6G available and have to modify this variable in the file to something smaller like 512m.

Reply:
Okay I understand the issue now. I think this is fine. Dropping the size is fine. There are/were some tests that needed 400+ MiB of GPU memory to work, and CUDF needs 500+ MiB so I rounded them both up to 1.5 GiB, but if you can manage with a lot less it is fine.  Just add a comment here so that it is clear it should not be used in CI tests and making a change is at your own risk.

Reply:
I added a WARNING message to the script